sklearn/linear_model/coordinate_descent.py

```diff
@@ -1537,15 +1556,17 @@ def __init__(self, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
 class MultiTaskElasticNet(Lasso):
     """Multi-task ElasticNet model trained with L1/L2 mixed-norm as regularizer
 
-    The optimization objective for MultiTaskElasticNet is::
+    The optimization objective for MultiTaskElasticNet is
 
-        (1 / (2 * n_samples)) * ||Y - XW||^Fro_2
-        + alpha * l1_ratio * ||W||_21
-        + 0.5 * alpha * (1 - l1_ratio) * ||W||_Fro^2
+    .. math::
+        \\frac{1}{2 * \\mathrm{n\\_samples}} * \\|Y - XW\\|^2_{\\mathrm{Fro}}
+        + \\alpha * \\mathrm{l1\\_ratio} * \\|W\\|_{2,1}
+        + 0.5 * \\alpha * (1 - \\mathrm{l1\\_ratio}) * \\|W\\|_{\\mathrm{Fro}}^2
```

Comment:
It was configured but I was confused because I got many more errors. After study, travis only evaluates the new code added.
If I do flake8 directly on sklearn/linear_model/coordinate_descent.py  I get:
```bash
sklearn/linear_model/coordinate_descent.py:74:80: E501 line too long (84 > 79 characters)
sklearn/linear_model/coordinate_descent.py:103:55: E128 continuation line under-indented for visual indent
sklearn/linear_model/coordinate_descent.py:104:55: E128 continuation line under-indented for visual indent
sklearn/linear_model/coordinate_descent.py:380:80: E501 line too long (83 > 79 characters)
sklearn/linear_model/coordinate_descent.py:546:80: E501 line too long (81 > 79 characters)
sklearn/linear_model/coordinate_descent.py:568:80: E501 line too long (84 > 79 characters)
sklearn/linear_model/coordinate_descent.py:734:80: E501 line too long (81 > 79 characters)
sklearn/linear_model/coordinate_descent.py:818:80: E501 line too long (84 > 79 characters)
sklearn/linear_model/coordinate_descent.py:1308:80: E501 line too long (84 > 79 characters)
sklearn/linear_model/coordinate_descent.py:1462:80: E501 line too long (84 > 79 characters)
sklearn/linear_model/coordinate_descent.py:1600:80: E501 line too long (84 > 79 characters)
sklearn/linear_model/coordinate_descent.py:1789:80: E501 line too long (84 > 79 characters)
sklearn/linear_model/coordinate_descent.py:1929:80: E501 line too long (84 > 79 characters)
sklearn/linear_model/coordinate_descent.py:2098:80: E501 line too long (84 > 79 characters)
```
In any case, now I discovered I can run the travis test locally so I can correct easier and earlier the mistakes.